### DISCUSSION TITLE
Error when editing project save

### DISCUSSION THREAD

**Discussion by waterrr** [dani-garcia/vaultwarden#3180] [Q&A] [answered]
I have upgraded to the latest, including the test version.
![3](https://user-images.githubusercontent.com/15225499/215037913-6554e95c-b037-4b29-8bfd-401d30892a3b.png)

Saving in the Google Chrome plugin will logout
In the mobile app it will prompt an error

Normal on the web

![1](https://user-images.githubusercontent.com/15225499/215037767-ce535675-8747-425c-8145-adf19e01fde7.png)
![2](https://user-images.githubusercontent.com/15225499/215037781-a7917371-2896-4501-8903-a8626bfa9453.png)

**Comment by waterrr** [comment]
I'm guessing there are some official(Bitwarden) changes or restrictions?

**Comment by BlackDex** [comment]
This has nothing to do with Bitwarden. I can use all the apps just fine without any issues, and i have all the latest versions currently available installed. That includes the `testing` tagged Vaultwarden image.

I suggest to fully logout of the clients and log back in. If that doesn't work, then is suggest to check the `/admin/diagnostics` page on your instance, and see if the date/time is in-sync and correct.

**Comment by waterrr** [comment]
> This has nothing to do with Bitwarden. I can use all the apps just fine without any issues, and i have all the latest versions currently available installed. That includes the `testing` tagged Vaultwarden image.
> 
> I suggest to fully logout of the clients and log back in. If that doesn't work, then is suggest to check the `/admin/diagnostics` page on your instance, and see if the date/time is in-sync and correct.

### Your environment (Generated via diagnostics page)
* Vaultwarden version: v1.27.0-9366e314
* Web-vault version: v2023.1.1
* OS/Arch: linux/x86_64
* Running within Docker: true (Base: Alpine)
* Environment settings overridden: false
* Uses a reverse proxy: true
* IP Header check: true (X-Real-IP)
* Internet access: true
* Internet access via a proxy: false
* DNS Check: true
* Time Check: true
* Domain Configuration Check: true
* HTTPS Check: true
* Database type: SQLite
* Database version: 3.39.2
* Clients used: 
* Reverse proxy and version: 
* Other relevant information: 

### Config (Generated via diagnostics page)
<details><summary>Show Running Config</summary>

**Environment settings which are overridden:** 


```json
{
  "_duo_akey": null,
  "_enable_duo": false,
  "_enable_email_2fa": false,
  "_enable_smtp": true,
  "_enable_yubico": true,
  "_icon_service_csp": "",
  "_icon_service_url": "",
  "_ip_header_enabled": true,
  "_smtp_img_src": "cid:",
  "admin_ratelimit_max_burst": 3,
  "admin_ratelimit_seconds": 300,
  "admin_token": "***",
  "allowed_iframe_ancestors": "",
  "attachments_folder": "data/attachments",
  "authenticator_disable_time_drift": false,
  "data_folder": "data",
  "database_conn_init": "",
  "database_max_conns": 10,
  "database_timeout": 30,
  "database_url": "******************",
  "db_connection_retries": 15,
  "disable_2fa_remember": false,
  "disable_admin_token": false,
  "disable_icon_download": false,
  "domain": "*****://***********",
  "domain_origin": "*****://***********",
  "domain_path": "",
  "domain_set": true,
  "duo_host": null,
  "duo_ikey": null,
  "duo_skey": null,
  "email_attempts_limit": 3,
  "email_expiration_time": 600,
  "email_token_size": 6,
  "emergency_access_allowed": true,
  "emergency_notification_reminder_schedule": "0 3 * * * *",
  "emergency_request_timeout_schedule": "0 7 * * * *",
  "enable_db_wal": true,
  "event_cleanup_schedule": "0 10 0 * * *",
  "events_days_retain": null,
  "extended_logging": true,
  "helo_name": null,
  "hibp_api_key": null,
  "icon_blacklist_non_global_ips": true,
  "icon_blacklist_regex": null,
  "icon_cache_folder": "data/icon_cache",
  "icon_cache_negttl": 259200,
  "icon_cache_ttl": 2592000,
  "icon_download_timeout": 10,
  "icon_redirect_code": 302,
  "icon_service": "internal",
  "incomplete_2fa_schedule": "30 * * * * *",
  "incomplete_2fa_time_limit": 3,
  "invitation_expiration_hours": 120,
  "invitation_org_name": "Vaultwarden",
  "invitations_allowed": true,
  "ip_header": "X-Real-IP",
  "job_poll_interval_ms": 30000,
  "log_file": null,
  "log_level": "Info",
  "log_timestamp_format": "%Y-%m-%d %H:%M:%S.%3f",
  "login_ratelimit_max_burst": 10,
  "login_ratelimit_seconds": 60,
  "org_attachment_limit": null,
  "org_creation_users": "",
  "org_events_enabled": false,
  "org_groups_enabled": false,
  "password_hints_allowed": true,
  "password_iterations": 600000,
  "reload_templates": false,
  "require_device_email": false,
  "rsa_key_filename": "data/rsa_key",
  "send_purge_schedule": "0 5 * * * *",
  "sends_allowed": true,
  "sends_folder": "data/sends",
  "show_password_hint": false,
  "signups_allowed": false,
  "signups_domains_whitelist": "",
  "signups_verify": false,
  "signups_verify_resend_limit": 6,
  "signups_verify_resend_time": 3600,
  "smtp_accept_invalid_certs": false,
  "smtp_accept_invalid_hostnames": false,
  "smtp_auth_mechanism": null,
  "smtp_debug": false,
  "smtp_embed_images": true,
  "smtp_explicit_tls": null,
  "smtp_from": "",
  "smtp_from_name": "Vaultwarden",
  "smtp_host": null,
  "smtp_password": null,
  "smtp_port": 587,
  "smtp_security": "starttls",
  "smtp_ssl": null,
  "smtp_timeout": 15,
  "smtp_username": null,
  "templates_folder": "data/templates",
  "tmp_folder": "data/tmp",
  "trash_auto_delete_days": null,
  "trash_purge_schedule": "0 5 0 * * *",
  "use_syslog": false,
  "user_attachment_limit": null,
  "web_vault_enabled": true,
  "web_vault_folder": "web-vault/",
  "websocket_address": "0.0.0.0",
  "websocket_enabled": false,
  "websocket_port": 3012,
  "yubico_client_id": null,
  "yubico_secret_key": null,
  "yubico_server": null
}
```
</details>

**Comment by waterrr** [comment]
ok fine. I found the cause, it was caused by Adguard error filtering.

**Comment by BlackDex** [comment, marked as answer]
Ah :) Good that you found it.
Yes Some Firewalls, Anti-Virus or Browser Extensions tend to break some other extensions or sites by blocking/filtering some headers.

**Reply by waterrr** [reply]
> Ah :) Good that you found it. Yes Some Firewalls, Anti-Virus or Browser Extensions tend to break some other extensions or sites by blocking/filtering some headers.

Adguard intercepts third-party authorization headers, just turn it off.
